index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  name: 'loader.js',
+
+  init: function() {
+    this.treePaths['vendor'] = 'lib';
+  },
+
+  included: function() {
+    this.app.import('vendor/loader.js', {
+      prepend: true
```

Comment:
is this sufficient, or should we make a special case option for the loader itself?